legal-api/src/legal_api/models/business.py

```diff
@@ -223,7 +223,7 @@ def validate_identifier(identifier: str) -> bool:
         except ValueError:
             return False
         # TODO This is not correct for entity types that are not Coops
-        if identifier[:-7] not in ('CP', 'XCP'):
+        if identifier[:-7] not in ('CP', 'XCP', 'NR', 'BC'):
```

Comment:
I moved the NR check up to line 213, since this breaks (doesn't allow valid NR value) when the NR is in the correct format "NR 1234567" with space. So you can remove "NR" from here and just leave "BC".

Reply:
should be resolved now